src/components/Modals/TransferManaModal/TransferManaModal.tsx

```diff
@@ -65,10 +68,21 @@ const TransferManaModal: React.FC<Props> = ({
     }
   }
 
+  const handleMax = () => {
+    if (network === Network.MATIC) {
+      setAmount(manaMatic)
+    } else {
+      setAmount(manaEth)
+    }
+  }
+
   useEffect(() => {
     onManaPrice()
   }, [])
 
+  const isDisabledByAmount =
+    network === Network.MATIC ? manaMatic < amount : manaEth < amount
```

Comment:
we should also add a check that the amount that you are trying to send/deposit/withdraw is greater than 0, otherwise I am able to do transfers and deposits of 0 MANA with an account that has no balance.

Reply:
Done! thanks!